core/sql/optimizer/OptimizerSimulator.cpp

```diff
@@ -278,9 +277,10 @@ OptimizerSimulator::OptimizerSimulator(CollHeap *heap)
     sysCallLogFilePath_[sc]=NULL;
     writeSysCallStream_[sc]=NULL;
   }
+  // Init log4cxx logging
+  CommonLogger::instance().initLog4cxx("log4cxx.trafodion.osim.config");
```

Comment:
Hi Hans, I know your concern, in this latest commit, I will only use fprintf for OSIM logging, actually, fprintf was prefered in the first place, and messages weren't shown in trafci client, OSIM logger is not going to impact logging of other modules.